charts/openstack-cluster/templates/_helpers.tpl

```diff
@@ -203,10 +203,11 @@ files:
       # This file is created by the capi-helm-chart to ensure that its parent directory exists
     owner: root:root
     permissions: "0644"
-  - path: /etc/containerd/config.toml
+  - path: /etc/containerd/config.d/containerd-certs.toml
     content: |
-      [plugins."io.containerd.grpc.v1.cri".registry]
-      config_path = "/etc/containerd/certs.d"
+      [plugins]
+        [plugins."io.containerd.grpc.v1.cri".registry]
+        config_path = "/etc/containerd/certs.d"
```

Comment:
We actually tried this, but it doesn't work because of this:

https://github.com/containerd/containerd/issues/5837

That is why we used `append: true` here, which clearly doesn't work in Ignition.

So this change will need to be reverted and we will need to think of another solution. It would be nice to be able to do this in Flatcar also (i.e. support registry mirrors).